diff --git a/config.def.h b/config.def.h
index 1c0b587..9ac858e 100644
--- a/config.def.h
+++ b/config.def.h
@@ -19,16 +19,17 @@ static const char *colors[][3]      = {
 };
 
 /* tagging */
-static const char *tags[] = { "1", "2", "3", "4", "5", "6", "7", "8", "9" };
+static const char *tags[] = { "1", "2", "3", "4", "5", "6" };
 
 static const Rule rules[] = {
 	/* xprop(1):
 	 *	WM_CLASS(STRING) = instance, class
 	 *	WM_NAME(STRING) = title
 	 */
-	/* class      instance    title       tags mask     isfloating   monitor */
-	{ "Gimp",     NULL,       NULL,       0,            1,           -1 },
-	{ "Firefox",  NULL,       NULL,       1 << 8,       0,           -1 },
+	/* class      		instance	title	tags mask	isfloating	monitor */
+	{ "Google-chrome",	NULL, 		NULL, 	1 << 1, 	0, 		-1 },
+	{ "Slack",		NULL,		NULL,	1 << 2,		0,		-1 },
+	{ "Keepassx",		NULL,		NULL,	1 << 3,		0,		-1 },
 };
 
 /* layout(s) */
